src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -178,7 +178,7 @@ private static void UploadDebugSymbols(IDiagnosticLogger logger, BuildTarget tar
 
             if (isMono)
             {
-                addFilesMatching($"{projectDir}/Temp", new[] { "**/UnityEngine.*.pdb", "**/Assembly-CSharp.pdb" });
```

Comment:
Are you sure the original files are still matched **for all supported versions of Unity**?
I remember some of these rules (not saying it relates to this one specifically) being very different for each version I've tested this on.

Maybe instead just add the new rule instead of removing the old one unless you've verified we're not losing useful files.

Reply:
I double checked and at the very least for Windows this works.
Line numbers on OSX and mono don't work right now. And that directory does not exist either. Checking for Linux.